setup.py

```diff
@@ -12,6 +12,9 @@
     long_description=readme,
     long_description_content_type="text/markdown",
     packages=find_namespace_packages(where="src"),
+    install_requires=[
+        'fire==0.4.0'
```

Comment:
Let's read the requirements file (`requirements.txt`) and pass it over.

Example:

```python
with open("requirements.txt", "r") as file:
    reqs = file.read().splitlines()
```